VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -147,20 +144,20 @@ public void onClick(View v) {
                 threadBody.setTopicId(allTopicsWithDepth.get(selectedTopicIndex).getDiscussionTopic().getIdentifier());
                 threadBody.setType(discussionQuestion);
 
-                createThread( threadBody );
-
+                addPostButton.setEnabled(false);
+                createThread(threadBody);
             }
         });
     }
 
-    protected void  createThread(ThreadBody threadBody){
-        if ( createThreadTask != null ){
+    protected void createThread(ThreadBody threadBody) {
+        if (createThreadTask != null) {
             createThreadTask.cancel(true);
         }
         createThreadTask = new CreateThreadTask(getActivity(), threadBody) {
             @Override
-            public void onSuccess(DiscussionThread  courseTopics) {
-               // addPostButton.setEnabled(true);
+            public void onSuccess(DiscussionThread courseTopics) {
+                // addPostButton.setEnabled(true);
```

Comment:
Sorry I hit auto-format and this file was a mess.

Anyway I removed that line in my other pull: f3c6963397158f00fdea4b3e4f157155L163